manifests/config.pp

```diff
@@ -259,7 +259,7 @@
   }
 
   $prometheus::server::collect_scrape_jobs.each |Hash $job_definition| {
-    if !stdlib::has_key($job_definition, 'job_name') {
+    if ! 'job_name' in $job_definition {
```

Comment:
```suggestion
    if ! ('job_name' in $job_definition) {
```

We had a bug caused by this already at https://github.com/voxpupuli/puppet-rsyslog/commit/a146191e6ec44f5a8163fff0887057eb0d08f6d3.